src/deployment/deploy.py

```diff
@@ -427,6 +427,14 @@ def try_sp_create() -> None:
 
         else:
             authorize_application(uuid.UUID(ONEFUZZ_CLI_APP), app.app_id)
+            if self.multi_tenant_domain:
+                authority = COMMON_AUTHORITY
+            else:
+                authority = cli_app.authority
+            self.cli_config = {
+                "client_id": cli_app.client_id,
```

Comment:
is this value different from `ONEFUZZ_CLI_AUTHORITY` when deploying outside of the redmond instance ?
Can you please share the value you see

Reply:
72f988bf-86f1-41af-91ab-2d7cd011db47 in redmond tenant and 975f013f-7f24-47e8-a7d3-abc4752bf346 in pme